### ISSUE TITLE
Ctrl+a inside of the "scale x" property text box does not select the text

### ISSUE THREAD

**Issue by jonesmz** [BearishSun/BansheeEngine#189]
Is the keyboard shortcut not configured for these?

![image](https://user-images.githubusercontent.com/755783/37437788-145a55ae-27bd-11e8-8f85-be6fed318f1e.png)

**Comment by BearishSun** [comment]
I don't think this is hooked up at the moment. I'll add it when I'm doing another pass on the editor, along with other missing usability features.

**Comment by christianclavet** [comment]
Hi, When I click inside the box the text is automatically selected. But you're right, CTRL-A is a expected behavior for a text/value field.
